recipes/pvl/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pvl" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "374e0c82bf53520066abc3f4b99b7cdd45f635519d28064add6f6b781279d491" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytz
+    - six
+  run:
+    - python
+    - pytz
+    - six
+
+test:
+  imports:
+    - pvl
+
+about:
+  home: http://github.com/planetarypy/pvl
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Python implementation of PVL (Parameter Value Language)'
+
+  description: |
+    PVL is a markup language, similar to xml, commonly employed for entries in
+    the Planetary Database System used by NASA to store mission data, among other
+    uses. This package supports both encoding a decoding a superset of PVL,
+    including the USGS Isis Cube Label and NASA PDS 3 Label dialects.
+  doc_url: http://pvl.readthedocs.org
+  dev_url: https://github.com/planetarypy/pvl
+
+extra:
+  recipe-maintainers:
+    - pbvarga1
+    - planetarypy
```

Comment:
`planetarypy` appears to be an organization not a user. If this is correct, please delete it.